docs/source/markdown/podman-image-scp.1.md

```diff
@@ -0,0 +1,84 @@
+% podman-image-scp(1)
+
+## NAME
+podman-image-scp - Securely copy an image from one host to another
+
+## SYNOPSIS
+**podman image scp** [*options*] *name*[:*tag*]
+
+## DESCRIPTION
+**podman image scp** saves an image to either **docker-archive**, **oci-archive**, **oci-dir** (directory with oci manifest type), or **docker-dir** (directory with v2s2 manifest type) on the local machine, and then copies the saved image to a remote host specified either by **--remote** globally, or by a connection name which has been added using **podman system connection add**. if neither **--remote** or a **HOSTNAME** has been specified, **image scp** will execute a local copy and save. default format is **docker-archive**. **podman image scp** writes to STDOUT by default and can be redirected to a file using the **output** flag. The **quiet** flag suppresses the output when set. **podman image scp** utilizes **podman save** and **podman load on the specified host or locally**.
+Note: `:` is used to specify the saved file location on the remote host. if not specified will default to the same path as specified for the **--output**
+
+**podman [GLOBAL OPTIONS]**
+
+**podman imaege scp [GLOBAL OPTIONS]**
+
+**podman image scp [OPTIONS] NAME[:TAG] HOSTNAME[:FILELOCATION]**
+
+## OPTIONS
+
+#### **--compress**
+
+Compress tarball image layers when pushing to a directory using the 'dir' transport. (default is same compression type, compressed or uncompressed, as source)
+Note: This flag can only be set when using the **dir** transport i.e --format=oci-dir or --format=docker-dir
+
+#### **--format**=*format*
+
+Save image to **docker-archive**, **oci-archive** (see `containers-transports(5)`), **oci-dir** (`oci` transport), or **docker-dir** (`dir` transport with v2s2 manifest type).
+```
+--format docker-archive
+--format oci-archive
+--format oci-dir
+--format docker-dir
+```
+
+#### **--multi-image-archive**, **-m**
```

Comment:
No need for this,  We don't need this complication.  Let's simplify this command out of the gate.